frontend/lib/ui/artefact_page/test_issues/test_issue_form.dart

```diff
@@ -0,0 +1,253 @@
+            Text(widget.formSubtitle),
+            const SizedBox(height: Spacing.level3),
+            VanillaTextInput(
+              label: 'Url',
+              controller: _urlController,
+              validator: (url) {
+                if (url == null || url.isEmpty) {
+                  return 'Must provide a bug/jira link to the issue';
+                }
+                if (Uri.tryParse(url) == null) {
+                  return 'Provided url is not valid';
+                }
+                return null;
+              },
+            ),
+            const SizedBox(height: Spacing.level3),
+            VanillaTextInput(
+              label: 'Description',
+              multiline: true,
+              controller: _descriptionController,
+              validator: (url) => url == null || url.isEmpty
+                  ? 'Must provide a description of the issue'
+                  : null,
+            ),
+            const SizedBox(height: Spacing.level4),
+            Row(
+              children: [
+                TextButton(
+                  onPressed: () => context.pop(),
+                  child: Text(
+                    'cancel',
+                    style: buttonFontStyle?.apply(color: Colors.grey),
+                  ),
+                ),
+                const Spacer(),
+                if (widget.onDelete != null)
+                  TextButton(
+                    onPressed: () {
+                      showDialog(
+                        context: context,
+                        builder: (_) => _DeleteTestIssueConfirmationDialog(
+                          onDelete: widget.onDelete!,
+                        ),
+                      );
+                    },
+                    child: Text(
+                      'delete',
+                      style: buttonFontStyle?.apply(color: Colors.red),
+                    ),
+                  ),
+                TextButton(
+                  onPressed: () {
+                    if (_formKey.currentState?.validate() == true) {
+                      widget.onSubmit(
+                        _urlController.text,
+                        _descriptionController.text,
+                      );
+                      context.pop();
+                    }
+                  },
+                  child: Text(
+                    'submit',
+                    style: buttonFontStyle?.apply(color: Colors.black),
+                  ),
+                ),
+              ],
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+}
+
+class _DeleteTestIssueConfirmationDialog extends StatelessWidget {
+  const _DeleteTestIssueConfirmationDialog({required this.onDelete});
+
+  final void Function() onDelete;
+
+  @override
+  Widget build(BuildContext context) {
+    return AlertDialog(
+      title: const Text('Are you sure you want to delete this issue?'),
+      content: const Text(
+        'Note that this will remove the issue for all related tests',
+      ),
```

Comment:
Can we just add the template id and the test case name, so to word it to something like:

Note that this will remove the issue for all tests with template id:X or test case name:Y